neo/SmartContract/StateReader.py

```diff
@@ -284,6 +284,9 @@ def Runtime_Notify(self, engine):
             payload
         )
 
+        message = payload[0] if len(payload) > 0 else payload
+        engine.write_log(str(message))
+
```

Comment:
FYI; This and the one below is added to make sure that we don't just see a 
```
14 SYSCALL
Neo.Runtime.Notify
```
but also see the string/argument it's printing in the log
```
14 SYSCALL
Neo.Runtime.Notify
b'Start main'
```